python-avd/pyavd/_eos_designs/shared_utils/utils.py

```diff
@@ -72,3 +72,7 @@ def get_merged_adapter_settings(self: SharedUtils, adapter_or_network_port_setti
         profile_name = adapter_or_network_port_settings.get("profile")
         adapter_profile = self.get_merged_port_profile(profile_name)
         return merge(adapter_profile, adapter_or_network_port_settings, list_merge="replace", destructive_merge=False)
+
+    def get_ip_from_ip_prefix(self: SharedUtils, ip_prefix: str) -> str:
```

Comment:
This should be moved to _utils and imported instead of having it in shared_utils.

Reply:
done